### PR TITLE
Test more recent PHP versions

### PR DESCRIPTION
**Pull request by ciaranmcnulty**
Replaces #136 and fixes the failing test

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-php: [5.6, 7.0, 7.1]
+php: [5.6, 7.0, 7.1, 7.2, 7.2]
```
[the remark this answers is not in the document]
Comment (ciaranmcnulty) [reply .travis.yml]: see #144

File: src/Behat/Gherkin/Node/TableNode.php [hunk 2]
```diff
@@ -44,6 +44,11 @@ public function __construct(array $table)
         $columnCount = null;
 
         foreach ($this->getRows() as $row) {
+
+            if (!is_array($row)) {
```
Comment (stof) [inline src/Behat/Gherkin/Node/TableNode.php]: this condition is now duplicated. You should removed the later one.

**Comment by ciaranmcnulty** [comment]
I should have waited for your review, @stof  - sorry

File: src/Behat/Gherkin/Node/TableNode.php (hunk 2 above)
Comment (guilliamxavier) [reply src/Behat/Gherkin/Node/TableNode.php]: see #164

**Pull request merged**